las/client.py

```diff
@@ -910,10 +910,31 @@ def create_model(
         :type width: int, optional
         :param height: The number of pixels to be used for the input image height of your model
         :type height: int, optional
-        :param preprocess_config: Specification of the processing steps prior to the prediction of an image
-        :type preprocess_config: dict
-        :param postprocess_config: Specification of the processing steps after the prediction of an image
-        :type postprocess_config: dict
+        :param preprocess_config: Pre processing configuration for predictions.
+            {
+                'autoRotate': True | False                          (optional)
+                'maxPages': 1 - 3                                   (optional)
+                'imageQuality': 'LOW' | 'HIGH'                      (optional)
+                'pages': List with up to 3 page-indices to process  (optional)
+            }
+            Examples:
+            {'pages': [0, 1, 2], 'autoRotate': True}
```

Comment:
Just to be explicit that 2 is not an upper limit.
```suggestion
            {'pages': [0, 1, 5], 'autoRotate': True}
```